front/src/components/TextEditor.js

```diff
@@ -0,0 +1,20 @@
+import './TextEditor.css';
+
+function TextEditor(props) {
```

Comment:
```suggestion
const TextEditor =({emptyName, ...})=>{
```
 Можно так и сразу props деструкторизировать

e.g:
<Component color="black" size="big" />

const Component = ({ color, size }) => ...